testsuite/Third Edition/AnonymousSpanImplicitDuration.ttml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<tt:tt xmlns:tt="http://www.w3.org/ns/ttml" xmlns:ttp="http://www.w3.org/ns/ttml#parameter" xmlns="http://www.w3.org/ns/ttml" xmlns:tts="http://www.w3.org/ns/ttml#styling" xmlns:ttm="http://www.w3.org/ns/ttml#metadata" ttp:profile="http://www.w3.org/ns/ttml/profile/imsc1/text" xml:lang="en">
+  <head>
+    <metadata>
+      <ttm:desc>Anonymous span implicit duration</ttm:desc>
+    </metadata>
+  </head>
+  <body>
+    <div>
+      <p begin="0s" end="1s" timeContainer="seq" tts:backgroundColor="blue">
+        No text should be displayed.
```

Comment:
Fine point: maybe change to "This text should not be displayed" so if it _is_ displayed that's clearly an error.